nixos/modules/services/monitoring/adguard-exporter.nix

```diff
@@ -0,0 +1,118 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let cfg = config.services.adguard-exporter;
+in {
+  options.services.adguard-exporter = {
+    enable = mkEnableOption "adguard-exporter";
+
+    interval = mkOption {
+      type = types.str;
+      default = "10s";
+      description = ''
+        Interval of time the exporter will fetch data from Adguard
```

Comment:
option descriptions (excluding the text of `mkEnableOption`) should end in a `.`, otherwise the generated manual looks weird.